frameworks/PHP/comet/benchmark_config.json

```diff
@@ -25,6 +25,8 @@
       "versus": "php"
     },
     "mysql": {
+      "json_url": "/json",
+      "plaintext_url": "/plaintext",
```

Comment:
Please, delete this 2 tests.
Are the same that with postgres.

Reply:
OK, done